blog_js/article.js

```diff
@@ -1,7 +1,5 @@
-//blog object
-var blog = {};
-var articleObjects = [];
 var newArticle;
```

Comment:
you don't need to declare newArticle here since you declare it in a function below